apps/vaporgui/Histo.h

```diff
@@ -20,42 +20,70 @@
 //
```

Comment:
The following code outputs a zero, while I suspect it should output a one.

```C++
#include <iostream>
#include "Histo.h"

int main()
{
    Histo h( 10, 0.0f, 1.0f, "haha", 1 );

    float v = 1.0f;
    h.addToBin( v );
    int idx = h.getBinIndexForValue( v );
    std::cout << h.getBinSize( idx ) << std::endl;
}
```

Reply:
Not my code.

Reply:
@StasJ is Histo.h going to be deprecated?

Reply:
I think it would be a good idea to replace it with something more robust

Reply:
Agreed. Let's leave it for future work